client/src/components/common/SprintModal/CreateSprintModal/index.tsx

```diff
@@ -1,11 +1,15 @@
 import React, { useState } from 'react';
+import DatePicker from 'react-datepicker';
```

Comment:
Let's move date picking from CreateSprintModal to EditSprintModal. Additionally, when editing sprint, let's make date picking disabled when toggle `isActive` is set to false.